src/identities.js

```diff
@@ -74,6 +77,9 @@ class Identities {
   }
 
   static async createIdentity (options = {}) {
+    const cacheKey = stringify(options)
```

Comment:
What happens if a `Keystore` instance is passed as `options.keystore`?

Reply:
Also, would it make more sense to use the actual `id` as the cache key?

Reply:
That was the first thing I tried, but `id` is not passed consistently in practice and waiting until after the identity is created defeats the purpose.

Deterministic stringify works but agree it's messy. Hashing the object could work?